src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -59,11 +60,12 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
     /**
      * Returns a LabList containing the list of strings given.
      */
-    public static LabList getLabSet(String... strings) {
+    @SafeVarargs
+    public static LabList getLabSet(Pair<String, String>... strings) {
         LabList ll = new LabList();
 
         ll.setLabs(Arrays.stream(strings)
-                .map(Lab::new)
+                .map(x -> (new Lab(x.getKey())).of(x.getValue()))
                 .collect(Collectors.toList()));
         return ll;
     }
```

Comment:
Only use `@SafeVarags` if you are sure that we cannot pollute the data. In this case after type erasure, `Pair` might not contain `String, String`. As of now, there should be no issues since we are only using it to generate test cases, but we might want to move this in the future for safety.

Reply:
Noted. However, do you think there are better ways instead of passing <code>Pair</code> to create labs with custom <code>LabStatus</code>? 

I am thinking of passing in a string <code>"1u"</code> to represent Lab1 UNSUBMITTED. That way we can prevent using <code>Pair</code> and avoid this issue, but on the other hand, it makes the creation of <code>TypicalPerson</code> harder to read